lunes_cms/cms/migrations/0005_document_created_by_link.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.2.13 on 2022-07-05 11:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("auth", "0013_group_icon"),
+        ("cms", "0004_feedback"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="document",
+            name="created_by_link",
+            field=models.ForeignKey(
+                blank=True,
+                max_length=255,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="auth.group",
+                verbose_name="created by",
+            ),
+        ),
+    ]
+
+
+def link_created_by(apps):
+    Document = apps.get_model("created_by", "Document")
+    Group = apps.get_model("created_by", "Group")
+    for document in Document.objects.all():
```

Comment:
```suggestion
    for document in Document.objects.filter(created_by__isnull=False):
```
We only have to set the link value for documents that actually have this value set.